File: papi/lista2/exercicio7.py
print("Aqui a área de um quadrado e o dobro desta área.")

while True:
    try:
        lado = float(input("Por favor favor informe o lado do quadrado em centímetros - use o ponto como separador decimal: "))
        
        if lado < 0:
            raise ValueError

        break
    except ValueError:
        print("Por favor informe um número válido e maior que zero.")

area = lado * lado
print("A área do quadrado em centímetros é", area)
print("O dobro da área do quadrado em centímetros é", area * 2)

File: papi/lista2/exercicio7.kt
package lista2

fun main() {
    var lado: Double
    while (true) {
        try {
            print("Digite o lado de um quadrado em centímetros, usando o ponto como separador decimal: ")
            val ladoString = readLine() 
            lado = ladoString!!.toDouble()
            break
        } catch (e: Exception) {
            println("Por favor, insira um valor válido para o lado do quadrado, usando o ponto como separador decimal")
        }
    }
    val area = lado * lado
    println("A área do quadrado de lado $lado centímetro(s) é $area centímetro(s) quadrado(s)")
    println("O dobro desta área é ${area * 2} centímetro(s) quadrado(s)")
}   